public/gameInfo.php

```diff
@@ -250,24 +242,10 @@ class='break-words whitespace-normal leading-normal'
     // Get user claim data
     if (isset($user) && $permissions >= Permissions::JuniorDeveloper) {
         $openTickets = countOpenTicketsByDev($user);
```

Comment:
Why not also move `$openTickets` into the blade? This whole block appears to be related to claims.

Reply:
Oof, I accidentally missed this one. Addressed in https://github.com/RetroAchievements/RAWeb/pull/1565/commits/8e6ee37ec631419be91d0fd8d602210cefbcf574.